custom.css

```diff
@@ -348,8 +345,74 @@ textarea:focus {
   outline: none;
 }
 
+@media only screen and (max-width: 767px) {
+  header ul {
+    display: none;
+  }
+
+  .menu-list {
+    gap: 1rem;
+    position: fixed;
+    top: 5rem;
+    right: 0;
+    background: transparent;
+    width: 100%;
+    height: 100%;
+    flex-direction: column;
+    z-index: 999;
+  }
+
+  .menu-list li {
+    width: 100%;
+    height: 6rem;
+  }
+
+  .menu-list li a {
+    width: 100%;
+    height: 100%;
+    padding-left: 1.3rem;
+    display: flex;
+    align-items: center;
+    transition: all 400ms ease;
+    color: white;
+  }
```

Comment:
- [ ] One of the requirements in this milestone is to follow Figma design in `(font, colors, images, text, margins).`. right now the nav-links in your design are not compatible with Figma design in font size, font-weight, and the margin between these navlinks. Kindly consider fixing these 3 things. you can take help from the inspect tab on figma website.
your design
![re](https://user-images.githubusercontent.com/96683843/231255986-402799a1-b3bd-4778-8da6-0ee7e6d4ea7c.jpg)

figma design
![qw](https://user-images.githubusercontent.com/96683843/231256011-2da203dc-3ecd-49a9-9c4a-8b371b695cec.jpg)